Adding --verbose DEBUG option fr iotedgectl

emmanuel-bv opened the pull request:
Adding --verbose DEBUG option fr iotedgectl and removing a comment on auto-cert-generate option which was misleading

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

emmanuel-bv committed:
Adding --verbose DEBUG option fr iotedgectl

Adding --verbose DEBUG option fr iotedgectl and removing a comment on auto-cert-generate option which was misleading
